development/otp-input-app/components/otp-input/ReuseExample/ReuseExample.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from "react";
+import { ReuseOTPInput } from "@locoworks/reusejs-react-otp-input";
+
+const ReuseExample = () => {
+  const [otp, setOtp] = useState("");
+  return (
+    <div className="flex flex-col items-center gap-x-3 justify-center py-10 mt-10 border rounded bg-gray-50">
+      <ReuseOTPInput
+        value={otp}
+        onChange={setOtp}
+        numInputs={5}
+        renderSeparator={null}
+        renderInput={(props: any) => <input {...props} />}
```

Comment:
Should this always be **input** only? Can we change this to anything other than input component? If yes what are the other posibilites? If not than can we move this inside the component? Or mention in docs that this line is mandatory?

Reply:
Sure @ShreyDhyani 

We can pass the wrapper divs or other wrapper components but we must and should use the input component inside it and pass the props to the input component. I will update the doc.